### PR TITLE
update to chameleon-3.0 and favicon

### PR DESCRIPTION
**Pull request by guoyunhe**
the mega menu is now a dropdown with app icons. icons changed to bootstrap-icons. some visual updates to navbar and search bars.

image

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -2,41 +2,56 @@
...
   <meta name="mobile-web-app-capable" content="yes" />
+  <meta name="theme-color" content="#73ba25" />
 
+  <!-- open graph meta -->
   <meta property="og:site_name" content="{{ site.title | escape }}" />
-  <meta property="og:title"
-    content="{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}" />
-  <meta property="og:type" content="article" />
-  {% if page.layout == 'post' %}
+  <meta property="og:title" content="{{ page.title | default: site.title | escape }}" />
```
Comment (hellcp) [inline _includes/head.html:38]: Semantically, short_title might work better here as the default

File: _includes/head.html [hunk 2]
```diff
@@ -77,7 +97,7 @@
   <!-- End Piwik Code -->
   {% endif %}
 
-  <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed/">
+  <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed/" />
```
Comment (hellcp) [inline _includes/head.html:100]: Some websites using chameleon don't have feeds at all

File: _includes/header.html [hunk 3]
```diff
@@ -1,17 +1,20 @@
 <nav class="navbar navbar-expand-md noprint{% if site.navbar_sticky %} sticky-top{% endif %}">
 
   <a class="navbar-brand" href="/">
-    {% include opensuse.svg %}
+    <img src="https://static.opensuse.org/favicon.svg" class="d-inline-block align-top" alt="🦎" title="openSUSE"
```
Comment (hellcp) [inline _includes/header.html:4]: Alt for our logo should always be `openSUSE`, this is what it semantically means

File: _includes/header.html [hunk 4]
```diff
@@ -73,11 +81,16 @@
...
+      xmlns="http://www.w3.org/2000/svg">
+      <path fill-rule="evenodd"
+        d="M1 2.5A1.5 1.5 0 0 1 2.5 1h3A1.5 1.5 0 0 1 7 2.5v3A1.5 1.5 0 0 1 5.5 7h-3A1.5 1.5 0 0 1 1 5.5v-3zM2.5 2a.5.5 0 0 0-.5.5v3a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-3zm6.5.5A1.5 1.5 0 0 1 10.5 1h3A1.5 1.5 0 0 1 15 2.5v3A1.5 1.5 0 0 1 13.5 7h-3A1.5 1.5 0 0 1 9 5.5v-3zm1.5-.5a.5.5 0 0 0-.5.5v3a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-3zM1 10.5A1.5 1.5 0 0 1 2.5 9h3A1.5 1.5 0 0 1 7 10.5v3A1.5 1.5 0 0 1 5.5 15h-3A1.5 1.5 0 0 1 1 13.5v-3zm1.5-.5a.5.5 0 0 0-.5.5v3a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-3zm6.5.5A1.5 1.5 0 0 1 10.5 9h3a1.5 1.5 0 0 1 1.5 1.5v3a1.5 1.5 0 0 1-1.5 1.5h-3A1.5 1.5 0 0 1 9 13.5v-3zm1.5-.5a.5.5 0 0 0-.5.5v3a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-3a.5.5 0 0 0-.5-.5h-3z">
+      </path>
     </svg>
   </button>
 
 </nav>
+
+<div id="megamenu" class="megamenu collapse"></div>
```
Comment (hellcp) [inline _includes/header.html:96]: This is no longer inserted with js?

Comment (guoyunhe) [reply _includes/header.html:96]: Not anymore. In this way we make it optional. The site can disable mega menu by remove these markups if it is unwanted.

File: _includes/head.html (hunk 1 above)
Comment (guoyunhe) [reply _includes/head.html:38]: `title` is `openSUSE News`, `short_title` is `News`. When sharing in third party sites, it is better to include `openSUSE News`, so the audience know which project it is. So I think we can keep `title`.

File: _includes/head.html (hunk 2 above)
Comment (guoyunhe) [reply _includes/head.html:100]: Now it is optional with `enable_feed` option.

File: _includes/head.html (hunk 1 above)
Comment (hellcp) [reply _includes/head.html:38]: openSUSE News is already a site_name though

**Comment by guoyunhe** [comment]
@hellcp any other thoughts? can we merge it?

**Pull request merged**